### PR TITLE
Added output directory option and fixed multiple atlas handling

### PR DESCRIPTION
**Pull request by danjgale**
This ended up being a dual pull request addressing #11 but also some fixes for providing multiple atlases. First, I added the option to specify the output directory (if not specified, the output directory defaults to the input path, as per the command line help). Then, the handling of multiple atlases was fixed following the integration of  #8 and #18. The code works nicely with specifying one or more atlases using argparse.

### CODE REVIEW

File: mni_atlas_reader/atlas_reader.py [hunk 1]
```diff
@@ -379,12 +389,30 @@ def create_output(filename, atlas, voxelThresh=2, clusterExtend=5,
         analysis
     probabilityThreshold : int
         Probability threshold for when using a probabilistic atlas
+    outDir : str or None
+        The full or relative path of an output directory created files are
```
Comment (miykael) [inline mni_atlas_reader/atlas_reader.py]: What do you mean with full or relative? How does it know which one to choose?
Would it make more sense to say something like "Path to the output directory. If None is specified, output will be stored in the same folder as the input files."

**Comment by miykael** [comment]
@rmarkello - can you check why travis is sad?

File: mni_atlas_reader/atlas_reader.py (hunk 1 above)
Comment (danjgale) [reply mni_atlas_reader/atlas_reader.py]: Either should work, as per the argparse help for `filename`. But I agree, I like your suggestion better

Comment (miykael) [reply mni_atlas_reader/atlas_reader.py]: Ah, ok. I don't have much `argparse` experience. If you want to change the description text. could you change that in your branch and send a new commit?

Comment (danjgale) [reply mni_atlas_reader/atlas_reader.py]: Done

**Comment by danjgale** [comment]
Just added a test for `create_output` now that we can specify directories. Should improve code coverage

**Comment by miykael** [comment]
That's great @danjgale. Thank you. I will merge this now.

**Pull request merged**